res/css/components/views/location/_ShareType.scss

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_ShareType {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+
+    padding: 60px $spacing-12 $spacing-32;
+
+    color: $primary-content;
+}
+
+.mx_ShareType_badge {
+    height: 60px;
+    width: 60px;
+    margin-bottom: $spacing-20;
+    background-color: $accent;
+    border-radius: 50%;
+}
+
+.mx_ShareType_heading {
+    padding-bottom: $spacing-32;
+    text-align: center;
+}
+
+.mx_ShareType_option {
+    @mixin ButtonResetDefault;
+
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+    padding: $spacing-8 $spacing-20;
+    margin-top: $spacing-12;
+
+    color: $primary-content;
+    border: 1px solid $quinary-content;
+    border-radius: 8px;
+
+    font-size: $font-15px;
+
+    &:hover, &:focus {
+        border-color: $accent;
+    }
+}
+
+.mx_ShareType_option-icon {
+    height: 40px;
+    width: 40px;
+    box-sizing: border-box;
+    margin-right: $spacing-12;
+    flex: 0 0 40px;
+    border-width: 2px;
+    border-style: solid;
+    border-radius: 50%;
+
+    &.Own {
+        // color is set by user color class
+        // generated from id
+        border-color: currentColor;
+    }
+
+    &.Live {
+        background-color: $location-live-color;
+        // TODO opacity
```

Comment:
presumably this will be done as part of live location sharing work in the future?